suite-native/discovery/src/discoveryThunks.ts

```diff
@@ -645,11 +653,15 @@ export const startDescriptorPreloadedDiscoveryThunk = createThunk(
                     }),
                 ).unwrap()) ?? [];
         }
+
+        // filter out cardano-type networks, if none of their derivations are available
         const networksFiltered = networksWithUnfinishedDiscovery.filter(
             network =>
                 network.networkType !== 'cardano' ||
-                (availableCardanoDerivations as string[]).includes(
-                    (network.accountType ?? NORMAL_ACCOUNT_TYPE) as string,
+                // first try normal derivation, if it's not available, try other derivations
+                availableCardanoDerivations.includes(NORMAL_ACCOUNT_TYPE) ||
+                filterUnavailableAccountTypes(network).some(({ accountType }: NetworkAccount) =>
+                    (availableCardanoDerivations as string[]).includes(accountType),
```

Comment:
If `filterUnavailableAccountTypes` leaves empty array, `.some()` will be false, which is expected - network will be filtered out :heavy_check_mark: 
But that probably won't happen, cause `filterUnavailableAccountTypes` filters out non-standard accounts...

Reply:
That's what L662 was for. But I used `normalizeNetworkAccounts` anyway in https://github.com/trezor/trezor-suite/pull/14281/commits/b92c24e0f3a100b1c17fa3ff3856b5b2c9240860, and it's much cleaner now I think :slightly_smiling_face: